setup/steps/configure_log_location.go

```diff
@@ -0,0 +1,163 @@
+package steps
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	survey "github.com/AlecAivazis/survey/v2"
+	"github.com/go-ini/ini"
+	"github.com/pganalyze/collector/setup/query"
+	"github.com/pganalyze/collector/setup/state"
+	s "github.com/pganalyze/collector/setup/state"
+)
+
+var ConfigureLogLocation = &s.Step{
+	Kind:        state.LogInsightsStep,
+	Description: "Determine log location",
+	Check: func(state *s.SetupState) (bool, error) {
+		return state.CurrentSection.HasKey("db_log_location"), nil
+	},
+	Run: func(state *s.SetupState) error {
+		guessedLogLocation, err := discoverLogLocation(state.CurrentSection, state.QueryRunner)
```

Comment:
We should not run the guessing logic if an explicit scripted input is set (`state.Inputs.Settings.DBLogLocation.Valid`)